lib/spack/spack/binary_distribution.py

```diff
@@ -770,19 +783,19 @@ def file_read_method(file_path):
         "--exclude",
         "*",
         "--include",
-        "*.spec.json.sig",
-        "--include",
-        "*.spec.json",
+        "*.manifest.json",
```

Comment:
This also syncs all of the other manifests, why not use the `"*.spec.manifest.json"` here?